components/layout/Sidebar.tsx

```
@@ -0,0 +1,106 @@
+                    </span>
+                  )}
+                </Button>
+              </Link>
+            )
+          })}
+      </nav>
+
+      {/* Footer: network + version */}
+      {!collapsed && (
+        <div className="mt-auto p-4 border-t border-gray-700/40 space-y-2">
+          <div className="flex items-center gap-2 text-xs text-gray-400" aria-label={`Network: ${environment.label}`}>
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)